### DISCUSSION TITLE
Custom email Rule

### DISCUSSION THREAD

**Discussion by ariesdg** [Jacobtims/filament-breezy#167] [Q&A] [answered]
How can i add custom email rule in Email Field under my profile?

**Comment by jeffgreco13** [comment, marked as answer]
I would recommend that you extend the MyProfile class with your own, then you can override the `getUpdateProfileFormSchema()` method to customize the form components as you wish:

https://github.com/jeffgreco13/filament-breezy/blob/6d5d73e5b55cbf9758207bd3ba5e58dfd4da4b9e/src/Pages/MyProfile.php#L60
